packages/mui-material/src/CardActions/CardActions.test.js

```diff
@@ -17,28 +17,27 @@ describe('<CardActions />', () => {
     skip: ['componentProp', 'componentsProp'],
   }));
 
-  it('should apply margin to all children but the first one', function test() {
-    if (window.navigator.userAgent.includes('jsdom')) {
-      this.skip();
-    }
+  it.skipIf(window.navigator.userAgent.includes('jsdom'))(
```

Comment:
This reflow is so annoying to review 😆

Reply:
yeah, best to disable whitespace in github files view